### PR TITLE
Env: update peer dependencies

### PR DESCRIPTION
**Pull request by Einere**
## Description

With current registered `dooboo-ui` in npm, users who use it with RN CLI need to install some additional dependent packages. (look below)

- @emotion/react
- @emotion/native
- react-native-vector-icons

The documents don't describe this issue, user had to install one by one looking errors.
So, this PR is adding the packages at `peerDependencies`.
Then, users don't need to additional installing the packages.
Please look linked issue for troubleshooting. 👏

## Related Issues

#74 

## Tests

I thinks this PR couldn't be tested by test codes. 🤔 

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: lib/package.json [hunk 1]
```diff
@@ -16,7 +16,10 @@
   "peerDependencies": {
     "react": ">=16.9",
     "react-native": ">=0.58",
-    "emotion": ">=11.0.0"
+    "emotion": ">=11.0.0",
+    "@emotion/react": "^11.4.1",
+    "@emotion/native": "^11.0.0",
+    "react-native-vector-icons": "^8.1.0"
```
Comment (hyochan) [inline lib/package.json:22]: How about adding more scopes to the versions like ">=11.0.0"?

Comment (hyochan) [reply lib/package.json:22]: For `react-native-vector-icons`, specifying it like below would be great.

```
"react-native-vector-icons": "*"
```

Comment (Einere) [reply lib/package.json:22]: ah, thank you for version checking. I didn't consider that. I will fix soon. 👏

Comment (Einere) [reply lib/package.json]: @hyochan
I fixed it. please check PR 🙇

File: lib/package.json [hunk 2]
```diff
@@ -17,9 +17,9 @@
     "react": ">=16.9",
     "react-native": ">=0.58",
     "emotion": ">=11.0.0",
-    "@emotion/react": "^11.4.1",
-    "@emotion/native": "^11.0.0",
-    "react-native-vector-icons": "^8.1.0"
+    "@emotion/react": ">=11.4.1",
```
Comment (hyochan) [inline lib/package.json:20]: I think this can be `>=11.0.1`

Comment (Einere) [reply lib/package.json:20]: oh, it's minimum version? I'll fix it!

**Review by hyochan — APPROVED** [review]
LGTM!

**Pull request merged**